vocabularies/Org.OData.Capabilities.V1.xml

```diff
@@ -570,11 +570,15 @@ supported:
           </Collection>
         </Annotation>
         <Property Name="Searchable" Type="Edm.Boolean" Nullable="false" DefaultValue="true">
-          <Annotation Term="Core.Description" String="$search is supported" />
+          <Annotation Term="Core.Description" String="$search is supported with the standard syntax [OData-URL, section 5.1.8.1](https://docs.oasis-open.org/odata/odata/v4.02/odata-v4.02-part2-url-conventions.html#SearchExpressions)" />
         </Property>
         <Property Name="UnsupportedExpressions" Type="Capabilities.SearchExpressions" Nullable="false" DefaultValue="none">
           <Annotation Term="Core.Description" String="Expressions not supported in $search" />
         </Property>
+        <Property Name="AlternativeSearchSyntax" Type="Edm.String" Nullable="true">
+          <Annotation Term="Core.IsURL" />
+          <Annotation Term="Core.Description" String="URI of the alternative $search syntax supported by the service" />

```

Comment:
```suggestion
          <Annotation Term="Core.Description" String="URL of the alternative $search syntax supported by the service" />
```

I'd prefer requiring a resolvable Locator here instead of a mere Identifier.